src/main/java/frc/robot/Constants.java

```diff
@@ -15,6 +15,9 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+  public static final double GRAVITY =
```

Comment:
What about Planck's constant? e?

Why do you have a GRAVITY constant?

Reply:
This is left over from when the arbitrary feed forward used physics instead of the WPILib class.